web/src/beta/features/ProjectSettings/innerPages/StorySettings/StorySettingsDetail.tsx

```diff
@@ -63,3 +73,16 @@ const StorySettingsDetail: React.FC<Props> = ({ settingsItem, onUpdateStory }) =
 };
 
 export default StorySettingsDetail;
+
+const FieldsWrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+  flex-direction: row;
+  gap: 8px;
+`;
+const SelectFieldWrapper = styled(SelectField)`
+  width: 100%;
+`;
+const ColorFieldWrapper = styled(ColorField)`
+  width: 100%;
+`;
```

Comment:
I feel like this shouldn't be necessary. The field's Property component that all fields share should already be taking up 100% width of the the parent (I think through flexing)...

Reply:
yes I thought that too, but the style was broken so I  make this change here because those components are used in alot of UIs and to avoid destroy any design.